src/context/thread_safe.rs

```diff
@@ -25,16 +25,16 @@ impl Deref for ContextGuard {
 /// A ThreadSafeContext can either be bound to a blocked client, or detached from any client.
 pub struct DetachedFromClient;
 
-pub struct ThreadSafeContext<B> {
+pub struct ThreadSafeContext<B: Send> {
     pub(crate) ctx: *mut raw::RedisModuleCtx,
 
     /// This field is only used implicitly by `Drop`, so avoid a compiler warning
     #[allow(dead_code)]
     blocked_client: B,
 }
 
-unsafe impl<B> Send for ThreadSafeContext<B> {}
-unsafe impl<B> Sync for ThreadSafeContext<B> {}
+unsafe impl<B: Send> Send for ThreadSafeContext<B> {}
```

Comment:
hmmm, its a good change, but did clippy catch this?

Reply:
yes it did, in pedantic or nursery I don't recall